pkg/rb2wf/converter.go

```diff
@@ -168,6 +168,10 @@ func (r *RunbookConverter) Convert(ctx context.Context, runbookSlug string) erro
 		return err
 	}
 
+	if strings.Contains(mainTsStr, manualFixPlaceholder) {
+		// TODO: include link to conversion tool docs with "see docs for more details."
+		logger.Warning("Some runbook features could not be converted directly to a task. Search for instances of '%s' and update the code accordingly.", manualFixPlaceholder)
```

Comment:
Copy tweak. Though could we just print an error message when it happens, so that we can print the template?

```suggestion
		logger.Warning("Some templates used features that are not supported in tasks. Search for instances of '%s' and update the code accordingly.", manualFixPlaceholder)
```